Create ui-c8y-1020-6-2-pan-to-marker-popup-to-make-it-always-visible.md (#2267)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1020-6-2-pan-to-marker-popup-to-make-it-always-visible.md

---
date: ""

> title: pan to marker popup to make it always visible

Author: The Claude AI suggested to use the following title:
```suggestion
title: Pan map to ensure marker popup is always visible
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-58511
version: 1020.6.2
---

> pan to marker popup to make it always visible

Author: The Claude AI suggested to use the following body:
```suggestion
In the Map component, when a marker popup was opened, it could sometimes be positioned outside the visible area of the map, making it difficult for users to view the popup content. With this change, the map now automatically pans to the position of the opened marker popup, ensuring that it is always fully visible within the map viewport. This improves the user experience when interacting with marker popups, as the relevant information is now consistently accessible and readable.
```
